public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -193,13 +175,14 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
       refId: target.refId,
       valueWithRefId: target.valueWithRefId,
       meta: {
-        /** Fix for showing of Prometheus results in Explore table.
+        /**
+         * Fix for showing of Prometheus results in Explore table.
          * We want to show result of instant query always in table and result of range query based on target.runAll;
          */
         preferredVisualisationType: target.instant ? 'table' : mixedQueries ? 'graph' : undefined,
       },
     };
-    const series = this.resultTransformer.transform(response, transformerOptions);
+    const series = transform(response, transformerOptions);
```

Comment:
So at this point this function is called only here and so i wonder if it makes sense to have the `processResult` function. It basically just maps some of the params and nothing else, maybe it would make sense to move that into the transform function and remove this additional step? What do you think?

Reply:
I like the idea.